pkgs/misc/vim-plugins/vim-plugin-names

```diff
@@ -64,6 +64,7 @@
 "github:wakatime/vim-wakatime"
 "github:wincent/command-t"
 "matchit.zip"
+"molokai"
```

Comment:
This plugin was in the database? Please change this line to `github:tomasr/molokai` to be definetly sure that everything will be ok :)